feat(content): The Successors - Enter the Frogtopus Aristocracy (#10296)
endless-sky/endless-sky: data/successors/successor 1 introduction.txt

			`	"Captain <last>. The High Houses have noted thy actions in their space, and wishing to make thee a stronger ally to their cause, have summoned thee to meet them in council. Thou art encouraged to make haste to the Uuoru-Kella system.`
			choice
				`	"Thank you for telling me. I'll prepare to travel there immediately."`
					accept
				`	"What do you mean 'meet them in council?'"`
			`	"Thy presence here is notable, Captain. Several representatives of the High Houses hath gathered to decide what is to be done with thee now that thou hast proved thy benevolence and usefulness. More than that, I cannot say."`
				accept
	on complete
		set "license: Successor"
		"reputation: Successor" += 5
		"reputation: House Chydiyi" += 5
		set "trusted by the successors"

> 		event "successors: trust 2"

Author: The event is meant to take place immediately.

			`	"I, for one, am pleased to hear of this, captain. The Houses can be jealous with their favors, but in thy case I think they are wise for dispensing them liberally. Ah," they say, interrupting themself, "would that I could stay, but my role here is yet to finish and I have another meeting still to entertain. Fair travels, captain, and may we meet again." They splay their arms wide in the same gesture you've seen many times before and you offer a short bow in return.`
				goto end
			label yours
			`	"I am pleased to hear of this, captain. I spoke at length about some of my human lingustic research, ah, thy tongues are fascinating items and make for truly interesting sounds... 'Twas also that I entertained a number of questions about thee thyself! Some young archivist from Kaatrij, I think? Truthfully, I sometimes confuse the New Houses with each other. Ah," they say, interrupting themself, "would that I could discuss it more, but my role here is yet to finish and I have another meeting still to entertain. Fair travels, captain, and may we meet again." They splay their arms wide in the same gesture you've seen many times before and you offer a short bow in return.`

> 			label end

Reviewer: Is this missing something?

			choice
				`	"Free?"`
					goto free
				`	"Thank you. I have no more questions."`
			`	"Thou would be wise, if thou possesses that rare art, to keep and guard thy back against thy new and treacherous 'friends.'`
			`	"Or don't. Another buried fool makes little difference."`
			`	With that, the Successor leaves, rising from the cargo containers and heading silently towards the spaceport.`
				decline
			label free
			`	"Free, or freer than Old and New permit. The People's Houses stand not for the rich and those of noble blood, but for the true and honest heirs of that which was."`
			`	The Successor looks over to the spaceport, before returning their attention to you for the last time.`
			`	"Thou would be wise, if thou possesses that rare art, to keep and guard thy back against thy new and treacherous 'friends.'`
			`	"Or don't. Another buried fool makes little difference."`
			`	With that, the Successor leaves, rising from the cargo containers and heading silently towards the spaceport.`

> 				decline

Reviewer: ```suggestion
				decline

```

			`	Sieasej and the scions turn their backs to you, and a line of soldiers converge from the walls to form a line separating you from them. The lead officer who escorted you inside motions with their weapon for you to leave.`
			`	As you exit, the door closes behind you, reforming into solid metal.`
			label end

substitutions
	"<tranq h2h>" "Pug staff"
	"<tranq h2h>" "Heliarch staff"
		has "outfit: Enforcer Riot Staff"

mission "Successors: First Contact 3"
	priority
	name "Capture an animal for House Sioeora"
	description `The scions of House Sioeora have instructed you to visit the planet <planet stopovers> and capture a "Kijra-riitaa" to safely return to them at <planet>. Return by <date> or they will consider you to have kidnapped Sieasej.`
	source "Staja-Kella-Oa"
	to offer
		has "Successors: First Contact 2: done"
		not "rejected: Successors: First Contact 2"
	deadline 30
	stopover "Raaqa-Uur-Kaav"
	destination "Staja-Kella-Oa"
	passengers 1
	blocked `You speak with Sieasej, who informs you that they require at least 1 free bunk to travel.`
	on offer

> 		conversation

Author: Let me know your thoughts on how I've applied `to display` and condition pruning to these missions. I don't return to the `questions` label for _every_ section because some of the sections have unique follow-up questions; I honestly think it would be more trouble than it's worth to try to implement these in a single master `questions` label because they are designed to be dependent on what was asked immediately prior. 
Nonetheless, this partial strategy does reduce redundancy somewhat.

# Copyright (c) 2024 by Daeridanii
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer 2: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
```

	on enter "Myruet-Kvelq"
		fail
		dialog
			`The system on the other side of the wormhole is the first one in days that your sensors do not respond to with alarms. Apart from the presence of the wormhole, the space here appears entirely normal.`

mission "Successors: First Contact 1"
	landing
	name "Go to the Vade-Staja system"
	description "The aliens in this area of space have indicated that you should travel to the easternmost inhabited system in their territory."
	source
		attributes successor
		not attributes uninhabited
	destination "Staja-Kella-Oa"
	clearance
	on offer
		conversation
			`After trudging through several shrouded and mostly-dead systems, you've finally found a planet with living inhabitants. As you enter the planet's upper atmosphere, your ship is surrounded by a trio of interceptors and thoroughly scanned. After a brief delay, the lead ship transmits an unintelligible hail to you and slowly proceeds to a spaceport complex floating on a large structure on the planet's coast, where the group corrals you onto one of the few fully-dry landing pads. As the minutes wear on, a small crowd begins to form around your landing site.`

> 			`	The aliens here remind you most of octopi: they are substantially taller than you, rising up off of the ground by several long suckered arms. Those standing on dry land are joined by several others who slither out of shallow, water-filled pools in the spaceport to come and watch you. Their skin glistens with mucous wetness and you can see it changing color and texture as they run their hourglass-shaped pupils over your person and your ship.`

Reviewer 3: I would probably side with the common usage over the technically correct one. It is likely that we'll see people asking to correct it either way but the common usage is likely going to feel better for most people.

> 			`	The aliens here remind you most of octopi: they are substantially taller than you, rising up off of the ground by several long suckered arms. Those standing on dry land are joined by several others who slither out of shallow, water-filled pools in the spaceport to come and watch you. Their skin glistens with mucous wetness and you can see it changing color and texture as they run their hourglass-shaped pupils over your person and your ship.`

Author: My plan is to reword this to hopefully remove the problem altogether, as mOctave suggested. (as soon as I come up with a rewording I like)

				flee

	npc accompany save
		government "House Sioeora"
		personality timid launching escort
		ship "Stolsaqra (Mission Escort)" "Kijet-i-khisa"
		ship "Stolsaqra (Mission Escort)" "Iyiq-sora"
		ship "Stolsaqra (Mission Escort)" "Seqra-saqra"

	on fail
		dialog `At least one of your escorts have been destroyed. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`

	on visit
		conversation
			`You have reached <planet>, but without your escorts, you are refused landing permission at the spaceport and must return to orbit until they arrive.`
				launch

	on complete
		event "label successor space"
		event "successors: first contact wait" 5
		log "Factions" "Successors" `The Successors are a society of aliens occupying a territory on the southeastern outer rim of the galaxy. They are the heirs of a large interstellar empire which collapsed at some point around fifteen hundred years ago, after exploring much of the galaxy, even as far north as Sol.`

> 		log "People" "Sieasej" `Sieasej is a Successor scholar of historical linguistics and member of the noble House Sioeora. Because of their area of study, they are one of the only Successors who can speak to and understand you without the benefit of a translation device.`

Author: The _intentional_ lore is that the Predecessors (Successors' ancestors) explored as far north as Sol in the ~1500s, did some spying (or "cultural observation") on humans then, but retreated back to their core territories and evacuated soon after as the space around their core worlds started to fall apart. Since then, they haven't gone any farther than one can go by hyperdrive from their space.
Sieasej is the equivalent of a medieval history scholar that learned how to speak 5th century Frankish. I like the idea that there are modern Successors who retain an interest in humans for the same reason that there are modern humans who retain an interest in dragons: they're both these weird charismatic megafauna that people talked about a long time ago, but **no**, there's no Successors spying on the Republic or whatnot---they don't know it exists unless FL tells them.

> 		log "People" "Sieasej" `Sieasej is a Successor scholar of historical linguistics and member of the noble House Sioeora. Because of their area of study, they are one of the only Successors who can speak to and understand you without the benefit of a translation device.`

Reviewer 4: Sorry, I thought I had deleted this comment after reading further...

Yes, your explanation makes perfect sense.

		"reputation: Successor" += 5
		"reputation: House Chydiyi" += 5
		"reputation: House Myurej" += 5
		"reputation: House Sioeora" += 5
		conversation
			`In a similar manner to the last time, you set down your ship on a landing pad at the spaceport of this planet, escorted by the interceptors. Differently to last time, however, you are shuttled to a small, out-of-the way area at a great distance from the main complex, where several uniformed individuals are gathered on the landing pad, standing at attention.`
			`	After waiting for several minutes, a shuttle lands on the pad beside yours and several individuals emerge from it. Two of them are moving more quickly than the rest, and the guards allow them to pass and approach you. The one on the left is a uniformed officer wearing what appears to be heavy body armor, shimmering metallic plates that sparkle in the light. On the right is a smaller alien, wrapped up in a dark, cocoon-like outfit from which their head and arms emerge. The officer says something to the smaller one, who quietly replies and then pulls from a pocket on their outfit a smooth sheet which they inspect several times before haltingly addressing you.`

> 			`	"Hark... noble visitor! We bid thee... welcome and good tidings." The two eye you closely as they wait for a response.`

Author: Making the dialogue the right degree of antiquated has been tricky to balance, as I like the thematic flavor it offers but don't want it to be too difficult to understand. I think these are very helpful substitutions and I'll try to work them in generally.

			choice
				`	"I see. What should I do now?"`
					goto whatnow
				`	"I see. What is House Sioeora?"`
					goto house
			label history
			`	"Twas a time in which the domains of our Predecessors did extend far to the north, and the reach of their curiosity and exploration even farther so. Life is precious and rare, doubly so that which is possessed of sapience and culture---noble in reason, infinite in faculty---as thine was and, we pray, still remains. So then, our Predecessors waited under inky cloaks to silently observe thee in thy glimmering development. And if 'twere not from the rest of the galaxy untimely ripp'd, we surely would have had the pleasure of watching thy species advance to its present point."`
			choice
				`	"What should I do now?"`
					goto whatnow
				`	"Is that how you know my language?"`
					goto language-short
				`	"What is House Sioeora?"`
					goto house
			label house

> 			`	Sieasej translates your query to the military officer this time, who considers for a moment before saying something back and Sieasej speaks again. "Sioeora is the noble house whose estate thou sees around thee. There are many such within the broader realm; the stars this side'f the wormhole, but 'tis Sioeora who has taken patronage on thee, for the keeping of our history is the... core of its purpose. 'Twere thee apprehended by the arms of another..." Sieasej gestures towards an interceptor landed on another pad "...thou surely would have languished without recognition as a creature from so distant in our past."`

Author: I believe these were --- previously before I realized no Unicode support; Unicode support is one update I'll definitely cheer for when the day comes.

					goto embark
				`	"Are you certain you're okay going with me?"`
					goto youokay
				`	"Is there anything I should watch out for?"`
					goto watchout
			label embark
			`	"I agree," Sieasej says.`
				accept
	on stopover
		conversation
			`Shortly after you enter the system, Sieasej enters your ship's cockpit and directs you to a location in the planet's southern hemisphere, on the edge of one of the smaller continents. "Here, captain," they say, "'twas a location for one of the old breeding grounds. When last our ships visited here, 'twas this location 'twere they noted most of the Kijran."`
			`	As you approach the coordinates to land, you see that it contains the remains of a substantial complex; over a dozen large buildings and several disused landing pads are present, connected by paths of smooth, untarnished metal. The materials show a curious quality of being both unblemished and overgrown; vines cling to otherwise-spotless walls and the various walkways, though often shaded by large plants, do not show any debris or detritus on their surfaces.`
			branch retrieve-tranqs
				or
					has "outfit: Enforcer Riot Staff"
					has "outfit: Pug Peacekeeping Staff"

> 			`	You set the <ship> down on one of the landing pads and exit onto the surface, though not before retrieving a net from storage. You are accompanied by Sieasej, who holds aloft a cylindrical device which they wave in the air for several seconds. "I read several calls of the species," they say. "We are fortunate, captain. We will not have to travel far." They wave their device around some more, making slower but wider motions, before they eventually speak again. "The nearest one lies to the west, about a hundred meters."`

Reviewer 4: What are "calls" supposed to mean here? Do you mean the actual noises they make? If so, "read" is a very confusing verb, although you might need to do a bit of restructuring to fit a better one.

				goto approach
			label retrieve-tranqs

> 			`	You set the <ship> down on one of the landing pads and exit onto the surface, though not before retrieving a net from storage, along with a <tranq h2h> from storage as an improvised tranquilizer weapon. You are accompanied by Sieasej, who holds aloft a cylindrical device which they wave in the air for several seconds. "I read several calls of the species," they say. "We are fortunate, captain. We will not have to travel far." They wave their device around some more, making slower but wider motions, before they eventually speak again. "The nearest one lies to the west, about a hundred meters."`

Reviewer 4: Same here

			label tranqed-alarmed
			`	You take aim and fire a shot from the <tranq h2h> at the animal, but it darts away at the last moment so that the blast only just grazes its body. It plummets to the ground shrieking, and while it soon goes quiet as the tranquilizer takes effect, the shriek is echoed by other animals in the area. Soon after those quiet as well, your ears are pierced by a loud, warbling wail from the distance.`
				goto escape
			label tranqed
			`	You take aim and fire a shot from the <tranq h2h> at the center of the animal. It briefly jumps to escape but soon plummets to the ground in a puff of leaves and dusty feathers.`
				goto bring-tranq
			label net
			`	You toss a net over the animal and it falls to the ground, where it shrieks for a duration before going quiet. The shriek is soon echoed by similar ones from other animals in the area, and once those go quiet as well, your ears are pierced by a loud, warbling wail in the distance.`
				goto escape
			label lure-1
			branch lure-1-success
				( random + has "Pookie, Part 5: offered" * 2 + has "Remnant: Face to Maw 1: completed" * 2 ) < 95
			`	The animal remains stationary for a moment before darting away.`

> 				goto lured-failed

Reviewer 4: Everything else is `lure-something`, you might want to change this label as well.

			label lure-1-success
			`	The animal remains stationary, examining you closely.`
			choice
				`	(Tranquilize it.)`
					goto tranq
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Catch it with a net.)`
					goto net
				`	(Slowly extend your hand.)`
					goto lure-2
			label lure-2
			branch lure-2-success
				( random + has "Pookie, Part 5: offered" * 2 + has "Remnant: Face to Maw 1: completed" * 2 ) < 95
			`	The animal inspects your hand for a moment before darting away.`
			label lured-failed
				`	(Tranquilize it.)`
					goto tranq
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Catch it with a net.)`
					goto net
			label lure-2-success
			`	The animal seems curious. It approaches your hand, sniffing it gently.`
			choice
				`	(Tranquilize it.)`
					goto tranq
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Catch it with a net.)`
					goto net
				`	(Offer it a morsel of food.)`
					goto lure-3

			label lure-3

> 			label lured

Reviewer 4: Is this a typo? You don't use the label `lured` anywhere else.

				goto escape
			label bring-tranq
			`	You scoop up the tranquilized animal, careful not to damage its extremities, and begin the walk back to your ship.`
				goto ambush
			label escape
			choice
				`	(Grab it and run!)`
			`	You scoop up the immobilized animal and begin to run back to your ship as the wailing behind you increases in pitch and intensity. As you and Sieasej dash through the foliage, the sunlight overhead is suddenly darkened by the shadow of something flying overhead.`
			choice
				`	(Keep running.)`
				`	(Try to shoot at whatever is above you.)`
					goto shoot-ambush
			`	Suddenly, the shadow disappears and the wailing stops. You keep running towards your ship, but can't shake the feeling that you are being watched.`
				goto ambush
			label shoot-ambush
			`	You draw your sidearm and look to see what's above you, but the shadow is suddenly gone and you are left staring up into the sunlight and leaves. The wailing has stopped as well; indeed, the whole area remains eerily quiet as you continue to run towards your ship.`
			label ambush

> 			`	When you arrive, however, you find that blocking your entry to the <ship> are two large creatures. Their glassy skin resembles that of the smaller creature you captured, but they are each several meters in size. Their feelers, rather than soft and downy, are whiplike and serrated and tipped with vicious-looking barbs that they thrust towards you and Sieasej upon seeing you. Their many eyes flick back and forth between you and the animal that you captured, and the two let out a loud wail in unison. Waves of red and blue run down Sieasej's body and you have to imagine it's an expression of fear.`

Reviewer 4: ```suggestion
			`	When you arrive, however, you find that blocking your entry to the <ship> are two large creatures. Their glassy skin resembles that of the smaller creature you captured, but they are each several meters in size. Their feelers, rather than soft and downy, are whiplike and serrated and tipped with vicious-looking barbs that they thrust towards you and Sieasej upon seeing you. Their many eyes flick back and forth between you and the animal that you captured, and the two let out a loud wail in unison. Waves of red and blue run down Sieasej's body, apparently an expression of fear.`
```

			choice
				`	(Run for it!)`
				`	(Shoot them with your sidearm.)`
					goto shoot

> 				`	(Order your ship to fire on them.)`

Reviewer 4: I like how you're involving the crew here... but what about an option to bring some of your crew with you? Or to pack something heavier than a sidearm? You don't necessarily need those options, but they could change this from a great mission to an exceptional one.

Author: Hmm, I like the idea of bringing crew especially; I'll have to think about what narrative role they could play since I don't just want them to be tagging along for the sake of it. My most immediate thought is that if the player chooses to bring crew, some of them might not come back. This mission is meant to leave a bad taste in the mouth if you go about it the most apparent way, and this could double down on that emotion.

> 				`	(Order your ship to fire on them.)`

Reviewer 4: ```suggestion
				`	(Order your crew to fire on them.)`
```

Unless your ship is sentient.

				goto end
			label released
			`	The waves of color running over Sieasej's body slowly fade once the animals are gone. "I confess I hadn't picked thee for such a tender soul, captain," they eventually say.`
			choice
				`	"I only did what I thought would get us back alive."`
					goto backalive
				`	"It wasn't right to hurt them."`
			`	"Ah," they say. "We are fortunate they thought the same of us."`
				goto released-2
			label backalive
			`	"Ah," they say. "Well I, for one, shall thank thee for thy prudence." After a delay, they add, "Even if Sioeora will not."`
			label released-2
			`	Sieasej briefly waves their device in the air unsuccessfully. "Nonetheless," they say, "thy act of mercy entails us failure. Under threat, the adults shall surely shroud the larvae, and so it is now here. It seems we must return with empty arms." They briefly undulate in a manner similar to a laugh. "I do not think I'd take another chase like that again regardless."`
			`	The two of you return to the <ship> and make preparations for takeoff so that you can, at the very least, return Sieasej to their people.`

> 				goto end

Reviewer 4: ```suggestion
```

			label end

	on complete
		conversation
			branch capture
				has "Successors FC3: captured kijra-pet"
			label release
			action
				"reputation: House Sioeora" = 0
				event "successors: released kijra-pet wait" 32
			`As before, you are directed to House Sioeora's private spaceport in the northern hemisphere, where a landing pad has already been cleared for your arrival. You and Sieasej are given fewer escorts than before, and you can't help but wonder if the absence of your quarry is particularly conspicuous to them. Soon enough, you once again arrive in the scions' chamber, where Riijaaora and the others stand waiting for you.`
			`	"Hark, captain <last>!" Riijaaora says, this time speaking with their own voice. The sounds feel warped and uneven, as if they are being rearranged midair. "Tell us, hast though brought the item of our desire?"`
			choice

> 				`	"No, I have not."`

Reviewer 4: Maybe add another option, just to make the choice make sense?

			`	An officer approaches you to offer you a data-sphere. "Within this," Riijaaora says, "is historical data for most of the worlds in our space. Combined with thy newfound ability of our language, thou should'st find the navigation of our worlds and cities more revealing and fruitful than before. Thou shall'st be permitted to gaze upon our technologies and vessels. Should'st the prospect interest thee, we are sure that the other Houses in council would be amenable to offering thee purchase rights, provided thou displays thy beneficence and utility to them as well. There are many minor tasks needing completion in our space; no doubt assisting with these would help thee ingratiate thyself with the others as thou hast with us." Riijaaora wiggles. "'Tis all, unless thou hast more questions thou needeth us resolve?"`
			choice
				`	"I have no questions."`
					goto end
				`	"Will being your ally bring me into conflict with the other Houses?"`
					goto others
				`	"What are you going to do to the animal?"`
					goto animal
			label others

> 			`	Riijaaora's skin blotches with colors. "Surely not, we think. None would make so brazen an insult so as to endanger the vassal of another. Thou may'st travel our space without fear."`

Reviewer 4: It occurs to me... since they speak in Shakespearean English, would they use the word "space" like this at all?

			`	"Captain <first> <last>," they say. Their voice sounds strange, like their words are being carried from a great distance on the wind; the sounds ripple and bend as they speak them.`
			choice
				`	"That's right."`
				`	"Who are you?"`
			`	"Hmm. Good. I am an envoy of the noble scions of House Kaatrij. I attest this myself as Saajret, of Somber Sejra." Saajret's skin erups with green splashes as they say this. "Thy patience has been much appreciated." They splay their arms in the same manner as when you were first greeted by the Successors, and the two others behind them do so as well. "Ah," they continue, "but thou surely must be curious as to why we've called thee here. Thy actions, knowing or not, have benefitted this noble house, and we would be craven fools to fail to recognize that. Prithee understand thee, Raaqa-Uur-Kaav was our world, not Sioeora's. So too, the Kijran were never theirs to hold or grasp for."`
			choice
				`	"So the Kijran belong to you?"`
					goto kijran
				`	"How is it that you can communicate with me?"`
					goto communicate
			label kijran

> 			`	Saajret shivers with texture. "More than just 'belong to us,' 'tis true. The Kijran---Kiitaa and Khora alike---are our creations, genetically engineered as the hybrids of a dozen precursor species. Why was it that the life of our world endured while the life of the rest perished through radiative erosion? It was the old House Kaatrij that strengthened their genomes against decay. That we were---had to leave them behind was one among many sorrows, and it is much better that thy quarry flies free than for another house to subject these children of ours to their whims.`

Reviewer 4: ```suggestion
			`	Saajret shivers with texture. "More than just 'belong to us,' 'tis true. The Kijran - Kiitaa and Khora alike - are our creations, genetically engineered as the hybrids of a dozen precursor species. Why was it that the life of our world endured while the life of the rest perished through radiative erosion? It was the old House Kaatrij that strengthened their genomes against decay. That we were - had to leave them behind was one among many sorrows, and it is much better that thy quarry flies free than for another house to subject these children of ours to their whims.`
```

			choice
				`	"How is it that you can communicate with me?`
					to display
						not "language: Successor"
					goto communicate
				`	"I see. What's next?"`
					to display
						has "language: Successor"
					goto end
				`	"Will your supporting me cause conflict with House Sioeora?"`
					to display
						has "language: Successor"
					goto sioeora
				`	"Can you tell me more about yourself?"`
					to display
						has "language: Successor"
					goto saajret
			label communicate

> 			`	"Ah, there is that matter. Much as the High Houses might prefer to believe that they---we act all within the veil of secrecy, we are all too intertwined to keep our activities hidden for long. The details of thy arrangement with Sioeora and its consequences were apparent to us soon after its conclusion, as will the details of this meeting eventually be apparent to the other houses in time." Saajret pauses for a moment to straighten some aspects of their clothing, before gesturing to a small wooden pin affixed to an outer layer of fabric. "We keep a number of these ancient translation devices, but very few for thine own language: this is the only one that we possess. I am told it is an art piece, commissioned for a performance of thy own Shakespeare's Tempest so that our Predecessors could experience the work in its original language. 'Tis far too old and unique to marry with our equipment; hence the need to contact thee in the language of another."`

Reviewer 4: ```suggestion
			`	"Ah, there is that matter. Much as the High Houses might prefer to believe that they - we act all within the veil of secrecy, we are all too intertwined to keep our activities hidden for long. The details of thy arrangement with Sioeora and its consequences were apparent to us soon after its conclusion, as will the details of this meeting eventually be apparent to the other houses in time." Saajret pauses for a moment to straighten some aspects of their clothing, before gesturing to a small wooden pin affixed to an outer layer of fabric. "We keep a number of these ancient translation devices, but very few for thine own language: this is the only one that we possess. I am told it is an art piece, commissioned for a performance of thy own Shakespeare's Tempest so that our Predecessors could experience the work in its original language. 'Tis far too old and unique to marry with our equipment; hence the need to contact thee in the language of another."`
```

			`	"House Kaatrij. House Chydiyi recognizes thy perspicacity and the value demonstrated by the captain, and supports thy proposal. The captain has proven themself a benevolent traveler in our space who we have every desire to make an ever more positive and useful independent element."`
			`	A third representative now takes the center position, addressing the two previous speakers.`
			`	"Houses Kaatrij and Chydiyi. House Sioeora proposes this alien be given no further rights nor gifts until they have proved themself neutral. The alien has already forsaken a task given by this House in favor of another: Kaatrij, whose ability to sway favor through brib'ry and deceit is quite clear. Should'st the alien be given purchase rights, their ability to offer wealth and power to individual factions will be even more formidable, and Sioeora stands strong in opposing this nascent corruption."`
			`	That representative then returns to the edge of the circle, where they continue to eye you with suspicion. There is a pause in the conversation.`
			choice
				`	(Remain silent.)`
					goto silent
				`	"I don't intend to take sides in your conflicts."`

> 					goto no

Reviewer 4: ```suggestion

```

			label end

mission "Successors: People's Houses, First Contact"
	landing
	source "Kasii-Tuur-Saqru"
	to offer
		has "known to the successors"
	on offer
		conversation
			`Compared to most of the spaceports you have been to in Successor space, this one is dingy, crowded, and uncomfortable. Most of the Successors you see look equally dirty and unhappy, and the majority of the few ships you've noticed landing are painted in a different, darker shade of red.`
			`	Would you like to ask someone here about this place?`
			choice
				`	(Yes.)`
				`	(Not yet.)`
					defer

> 			`	You approach one of the crew members disembarking from one of the red-painted ships, which, as you draw closer, you see bears several large burns and scars.`

Reviewer 4: This is a little too much in the way of commas, and wordy. How about
```suggestion
			`	You approach one of the crew members disembarking from a red-painted ship which bears a spectacular number of burn marks and scars.`
```

			`	The two once again proceed slowly and carefully, the speaker briefly conversing with the military officer before each phrase. "I am Sieasej... of House Sioeora among the Successors. Thou'rt a human, art thou not?"`
			choice
				`	"Yes, I am."`
				`	"Who else would I be?"`
			`	This sparks a wave of color among the gathered members, most of all Sieasej, who spends several seconds in a vibrant shade of orange before returning to their resting hue. "Then i'tis confirmed. 'Tis a cause for much excitement, <last>, for thou'rt the first human met by us in fifteen hundred years, since the time before thy species had even left the bounds o' thy own planet's gravity. House Sioeora is honored to make thy acquaintance, as am I." The members of the group splay their free arms towards you and glisten with ripples of blue across their skin. You perform a short bow in return.`
			choice
				`	"What should I do now?"`
					goto whatnow
				`	"How do you know my language?"`
					goto language
				`	"How could you have met humans fifteen hundred years ago?"`
					goto history
				`	"What is House Sioeora?"`
					goto house
			label language

> 			`	"When once our Predecessors did explore and look upon the life of the galaxy beyond their borders, they were careful to record its details and its subtleties, for life is precious and rare. And doubly so that which is possessed of sapience and culture - noble in reason, infinite in faculty - as the nascent humanity was and, we pray, still remains. So then, our Predecessors waited under inky cloaks to silently observe thee in thy glimmering development. And if 'twere not from the rest of the galaxy untimely ripp'd, we surely would have had the pleasure of watching thy species advance to its present point."` 

Reviewer: ```suggestion
			`	"When once our Predecessors did explore and look upon the life of the galaxy beyond their borders, they were careful to record its details and its subtleties, for life is precious and rare. And doubly so that which is possessed of sapience and culture - noble in reason, infinite in faculty - as the nascent humanity was and, we pray, still remains. So then, our Predecessors waited under inky cloaks to silently observe thee in thy glimmering development. And if 'twere not from the rest of the galaxy untimely ripp'd, we surely would have had the pleasure of watching thy species advance to its present point."`
```

				"reputation: House Myurej" += 5
				"reputation: House Aqrabe" += 10
				"reputation: House Kaatrij" += 10
				"reputation: House Seineq" += 10
				"reputation: People's Houses" += 10

mission "Successors: Kaatrij Monoglot Prompt"
	landing
	invisible
	to offer
		has "event: successors: released kijra-pet wait"
		and
			not "language: Hai"
			not "language: Coalition"
		random < 30
	on offer
		conversation
			`Shortly after you set the <ship> down on <origin>, you notice a faint trilling coming from your cabin on board. After rummaging through some of your belongings, you discover that it is coming from a Successor data-sphere, which is vibrating intensely. You pick it up, and it projects a message in a strange-looking alien script that you unfortunately do not recognize. If you ever acquired some alien translation equipment, you might be able to decode it.`
			decline

substitutions
	"<successor patron>" "Sioeora"
	"<successor patron>" "Kaatrij"
		has "completed: Successors: Kaatrij First Contact 1"
	"<successor opponent>" "Kaatrij"
	"<successor opponent>" "Sioeora"
		has "completed: Successors: Kaatrij First Contact 1"

mission "Successors: Trusted"
	landing

> 	name "Go to the Uuoru-Kella system to meet with the High Houses"

Reviewer: Too long.
```suggestion
	name "Meet the High Houses"
```

			`	As before, your landing site has been surrounded by a cadre of uniformed individuals, but here there must be nearly a hundred standing at attention on the pad's perimeter, of which most appear heavily armed. Once again upon disembarking, a lead officer approaches you, this time accompanied by a group of eight others who follow them in a tight formation. The leader motions with their arm towards the center of the complex as if to instruct you to go.`
			choice
				`	(Go with them.)`
					goto inside
				`	"What is this place?"`
			`	The lead individual looks you over and shuffles their arms somewhat before repeating their motion towards the center of the complex.`
			choice
				`	(Go with them.)`
					goto inside
				`	"Is this where the scions are?"`
			`	The leader says something to one of the others before returning their attention to you and gesturing again, this time with their weapon.`
			choice

> 				`	(Go).`

Reviewer: ```suggestion
				`	(Go.)`
```

			label questions
			choice
				`	"What should I do now?"`
					goto whatnow
				`	"How do you know my language?"`
					to display
						not "Successors: language"
					goto language
				`	"How could you have met humans fifteen hundred years ago?"`
					to display
						not "Successors: history"
					goto history
				`	"What is House Sioeora?"`
					to display
						not "Successors: house"
					goto house
			label language
			action
				set "Successors: language"
				set "Successors: history"
			`	"T'was a time in which our Predecessors' home domains reach'd farther to the north, and farther still the reach of their intention to explore. For life, it is a rare and precious thing, and doubly so that life which is possess'd of sapience and culture, that's noble in reason, infinite in faculty, as thine. So then, our Predecessors waited under cloak of dark to silently observe thee in thy glimmering development. And if 'twere not from the rest of the stars so untimely ripp'd, we surely would have had the pleasure t'watch thy kind advance to what's its present point."`

> 			`	Sieasej pauses for a moment, wriggling under their cloak. "I am a scholar of some olden languages," they say, "and of the worlds our Predecessors met so long ago. I'm one of very few who's learned thy tongue, and others, at least as 'twas gone fifteen hundred years ago."`

Reviewer: This feels rather awkward. Maybe do this to start?
```suggestion
			`	Sieasej pauses for a moment, wriggling under their cloak. "I am a scholar of some olden languages," they say, "and of the worlds our Predecessors met so long ago. I happen to be one of very few who's learned thy tongue, and others, at least as 'twas gone fifteen hundred years ago."`
```

Author: You're right that the sentence was awkward; I've simplified it overall rather than add more words.

Reviewer: Looks good.

			choice
				`	"What should I do now?"`
					goto whatnow
				`	"What do you mean that you were ripped from the galaxy?"`	# Define choices uniquely when they contain a follow-up question not in label "questions."
					to display
						not "Successors: ripped"
					goto ripped
				`	"What is House Sioeora?"`
					to display
						not "Successors: house"
					goto house
			label language-short
			action
				set "Successors: language"

> 			`	"T'is true," they say. "I am a scholar of some olden languages, and of the worlds our Predecessors met so long ago. I'm one of very few who's learned thy tongue, and others, at least as 'twas gone fifteen hundred years ago."`

Reviewer: ```suggestion
			`	"T'is true," they say. "I am a scholar of some olden languages, and of the worlds our Predecessors met so long ago. I happen to be one of very few who's learned thy tongue, and others, at least as 'twas gone fifteen hundred years ago."`
```

			choice
				`	"I only did what I thought would get us back alive."`
					goto backalive
				`	"It wasn't right to hurt them."`
			`	"Ah," they say. "We are fortunate they thought the same of us."`
				goto released-2
			label backalive
			`	"Ah," they say. "Well I, for one, shall thank thee for thy prudence." After a delay, they add, "Even if Sioeora will not."`
			label released-2
			`	Sieasej briefly waves their device in the air unsuccessfully. "Nonetheless," they say, "thine act of mercy entails us failure. Under threat, the adults shall surely shroud the larvae, and so it is now here. It seems we must return with empty arms." They briefly undulate in a manner similar to a laugh. "I do not think I'd take another chase like that again regardless."`
			`	The two of you return to the <ship> and make preparations for takeoff so that you can, at the very least, return Sieasej to their people.`
			label end

	on fail
		dialog `You failed to return Sieasej to their people in time. If you wish to continue the Successors' story, revert to a prior save.`

> 		"reputation: House Sioeora" = -999

Author: -999 was just the placeholder value I chose to indicate that they hate you permanently. There's a similar thing where the Gegno Genocide Defense mission sets all Gegno reps to -1000; I suppose I just found -999 stood out more against the rest.

That being said, it probably makes sense to change the People's Houses (Hostile) government to use a less extreme negative value so that the player could conceivably work their way into their good graces.

		"reputation: Successor" = 0
		"reputation: House Chydiyi" = 0

> 		"reputation: House Myurej" = 0

Author: The reputations with the other Houses are still at 0 at this point so I didn't see a reason to change them.

	on enter "Myruet-Kvelq"
		fail
		dialog
			`The system on the other side of the wormhole is the first one in days that your sensors have not responded to with alarms. Apart from the presence of the wormhole, the space here appears entirely normal.`

mission "Successors: First Contact 1"
	landing
	name "Go to the Vade-Staja system"
	description "The aliens in this area of space have indicated that you should travel to the easternmost inhabited system in their territory."
	source
		attributes successor
		not attributes uninhabited
	destination "Staja-Kella-Oa"
	clearance
	on offer
		conversation
			`After trudging through several shrouded and mostly-dead systems, you've finally found a planet with living inhabitants. As you enter the planet's upper atmosphere, your ship is surrounded by a trio of interceptors and thoroughly scanned. Soon after, the lead ship transmits a hail to you in a sharply flowing alien language before proceeding to a spaceport complex floating on a large structure on the planet's coast. The group corrals you onto one of the few fully-dry landing pads, and as the minutes wear on, a small crowd begins to form around you and your ship.`

> 			`	The predominant aliens here are enormous octopods that rise up off of the ground on their long, suckered arms. On average, they stand about three meters tall even with their arms coiled beneath them. Those standing on dry land are joined by others who slither out of a network of water-filled pools and channels to come and watch you. Their skin glistens with mucous wetness and you can see it changing color and texture as they run their hourglass-shaped pupils over your person and your ship.`

Reviewer 5: `rise up from the ground`, perhaps?

			`	After a couple of minutes of activity within the crowd, one uniformed and especially vibrant individual breaks away from the rest and begins to move towards you, gliding gently and silently over the ground. They say something the same strange language, shimmering with waves of color, and then wait expectantly for you to reply.`
			choice
				`	(Say nothing.)`
					goto map
				`	"Hello, I'm Captain <first> <last>."`
					goto introduce
				`	"I come in peace."`
					goto introduce
				`	(Shoot them.)`
			`	You draw your weapon and fire a shot directly at the center of the uniformed alien, but it does not manage to even scorch the fabric of their clothing. Before you have time to fire a second time, you hear, see, and feel an unfathomably loud noise that pushes you back and leaves you dead before your body even makes contact with the ground.`
				die
			label introduce
			`	The alien shuffles their arms in place and leans their face towards yours, repeating their earlier statement, this time more slowly and at a lower pitch. When you once again fail to give a satisfactory response, they pull away.`
				goto map
			label map

> 			`	After a few more moments of silence, the alien grunts in a gesture something like a sigh of disappointment and then turns away and says something to another uniformed individual waiting on the edges of the now-substantial crowd. The other's skin flashes a bright blue and then they leave, gliding towards one of the interceptors now parked alongside your ship.`

Author: I like it; I think for the last one `before they leave` would be better so that it can't be misinterpreted as their skin being the thing that leaves.

				set "rejected: Successors: First Contact 2"
				"reputation: House Sioeora" = 0
			`	A ripple of color runs up the body of Sieasej and several of the other Successors. The scion gives their response slowly. "Then our act of grace today shall be t'allow thee to depart this noble place still possessed o'thy freedom and belongings. Thou shalt receive no further aid nor comfort from this House, and shalt be banished at the pleasure of its scions from whatever worlds lie under our dominion."`
			`	Sieasej and the scions turn their backs to you, and a line of soldiers converge from the walls to form a line separating you from them. The lead officer who escorted you inside motions with their weapon for you to leave.`
			`	As you exit, the door closes behind you, reforming into solid metal.`
			label end

substitutions
	"<tranq h2h>" "Pug staff"
	"<tranq h2h>" "Heliarch staff"
		has "outfit: Enforcer Riot Staff"
	"<tranq h2h long>" "Pug Peacekeeping Staff"
	"<tranq h2h long>" "Enforcer Riot Staff"

> 		has "outfit: Enforcer Riot Staff"

Author: How do you mean? I'm not sure I understand.

> 		has "outfit: Enforcer Riot Staff"

Reviewer 6: You can define substitutions inside a mission:
```
mission <name>
		...
	substitutions
		<text> <replacement>
			[<condition set>]
```
These substitutions only exist within this mission.

> Beginning with v.0.9.15, this specifies custom text replacements that apply only to the text of the mission they're defined within. Substitutions defined within a mission take precedence over global substitutions and are overtaken in precedence by [hardcoded text replacements](https://github.com/endless-sky/endless-sky/wiki/CreatingMissions#text-replacements). For more information on custom text replacements, see the [creating substitutions](https://github.com/endless-sky/endless-sky/wiki/CreatingSubstitutions) page.

				goto questions
			label jumpdrives
				set "Successors: jumpdrives"
			`	"Remember, Captain, our Predecessors did ply the stars for ten thousand years before we last set eyes on thine own race, who then had only just discovered industry. Drives like the one within thy ship have been known to us for long into thy past."`
			choice
				`	"We'd better get going."`
					goto embark
				`	"Do the scions have their own jump drives?"`
					goto warning
				`	"What more can you tell me about the Kijra-pets?"`
					goto kijra-pet
				`	"Are you certain you're okay going with me?"`
					goto youokay
				`	"Is there anything I should watch out for?"`
					goto watchout
			label warning
				set "Successors: warning"
			`	"That is not for me to know, nor, I think, for thee."`
				goto questions
			label embark
			action
				clear "Successors: jumpdrives"
				clear "Successors: kijra-pet"
				clear "Successors: knowledge"
				clear "Successors: warning"
				clear "Successors: watchout"
				clear "Successors: youokay"
			`	"I agree," Sieasej says. "We have no time to squander."`
				accept

> 	on stopover

Reviewer 6: This is way over-engineered.

			label approach
			choice
				`	(Approach the location.)`
					goto encounter
				`	"You know a lot about tracking for a scholar."`
			`	"Ah," Sieasej says, pausing for a moment. "A product of some training I've not used in many years. Perhaps Sioeora's forward-thinking as much as they'd have us believe."`
			label encounter
			`	You make your way along one of the walkways to the west, brushing aside the enormous leaves of the overgrowing plants. Sieasej continues to wave their triangulation device in the air and quietly relays your progress; as you get closer, you hear a gentle scuffling that you realize precedes every detection by the device. After making your way through one particularly-dense wall of flora, you spot your target: a glassy-winged animal perched in a plant over the walkway's edge, probing the branch it stands on with long feelers extending from its head.`
			choice
				`	(Try to tranquilize it.)`
					goto tranq
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Try to catch it with a net.)`
					goto net
				`	(Stay still.)`
					goto lure-1
			label tranq
			branch tranqed
				random < 70

> 			label tranqed-alarmed

Reviewer 6: ```suggestion
```

				goto escape
			label tranqed
			`	You take aim and fire a blast from the <tranq h2h> at the center of the animal. It briefly jumps to escape but soon plummets to the ground in a puff of leaves and dusty feathers.`
				goto bring-tranq
			label net
			`	You toss a net over the animal and it falls to the ground, where it shrieks for a duration before going quiet. The shriek is soon echoed by similar ones from other animals in the area, and once those go quiet as well, your ears are pierced by a loud, warbling wail in the distance.`
				goto escape
			label lure-1
			branch lure-1-success
				random < ( 95 + has "Pookie, Part 5: offered" * 2 + has "Remnant: Face to Maw 1: completed" * 2 )
			`	The animal remains stationary for a moment before darting away.`
				goto lure-failed
			label lure-1-success
			`	The animal remains stationary, examining you closely.`
			choice
				`	(Tranquilize it.)`
					goto tranq
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Catch it with a net.)`
					goto net
				`	(Slowly extend your hand.)`

> 					goto lure-2
> 			label lure-2

Reviewer 6: ```suggestion
```

			branch lure-2-success
				random < ( 95 + has "Pookie, Part 5: offered" * 2 + has "Remnant: Face to Maw 1: completed" * 2 )
			`	The animal inspects your hand for a moment before darting away.`
			label lure-failed
				`	(Tranquilize it.)`
					goto tranq
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Catch it with a net.)`
					goto net
			label lure-2-success
			`	The animal seems curious. It approaches your hand, sniffing it gently.`
			choice
				`	(Tranquilize it.)`
					goto tranq
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Catch it with a net.)`
					goto net
				`	(Offer it a morsel of food.)`
			`	You retrieve a morsel of food and offer it to the animal, which sniffs it gently before starting to eat some.`
			choice
				`	(Tranquilize it now.)`
					goto tranq-friends
					to display
						or
							has "outfit: Enforcer Riot Staff"
							has "outfit: Pug Peacekeeping Staff"
				`	(Catch it with a net.)`
					goto net-friends-alarmed
				`	(Try to bring it back with food alone.)`

> 					goto friends

Reviewer 6: ```suggestion
```

				goto ambush
			label shoot-ambush
			`	You draw your sidearm and look to see what's above you, but the shadow is suddenly gone and you are left staring up into the sunlight and leaves. The wailing has stopped as well; indeed, the whole area remains eerily quiet as you continue to run towards your ship.`
			label ambush
			`	When you arrive, however, you find that blocking your entry to the <ship> are two large creatures. Their glassy skin resembles that of the smaller creature you captured, but they are each several meters in size. Their feelers, rather than soft and downy, are whiplike and serrated and tipped with vicious-looking barbs that they thrust towards you and Sieasej upon noticing your presence. Their many eyes flick back and forth between you and the animal that you captured, and the two let out a loud wail in unison. Waves of red and blue run down Sieasej's body, probably an expression of fear.`
			choice
				`	(Run for it!)`
				`	(Shoot them with your sidearm.)`
					goto shoot
				`	(Order your crew to fire a turret at them.)`
					to display

> 						"flagship base attribute: turret mounts" > "flagship attribute: turret mounts"

Reviewer 6: Not every turret is hostile; AM might not be useful. Also, turrets like a Firestorm Battery would obliterate everything nearby, maybe even your ship.

Author: I got the idea for this action from the Bunrodea first contact mission, where the same condition is used to check if First Last can disable a Bunrodea anti-repulsor vehicle on the ground. It's a bit of an abstraction in both cases, but I like the additional option. Perhaps I should be more vague about the weapon's effects?

			`	As you return to the landing pad, you notice that while your ship has been refueled, the pad itself is slowly being flooded in a none-too-subtle gesture that encourages you to leave.`
				goto exit

			label capture
			action
				set "known to the successors"
				set "language: Successor"
				event "successors: trust 1"
				"reputation: Successor" += 5
				"reputation: House Chydiyi" += 5
				"reputation: House Myurej" += 5
				"reputation: House Sioeora" += 5
				"reputation: House Aqrabe" += 10
				"reputation: House Seineq" += 10
				"reputation: People's Houses" += 10
				log "Captured and returned the animal House Sioeora requested that I bring to them. Was rewarded with a translation device and historical data that allows me to land on most Successor worlds. Was told that assisting in jobs and missions for the Successors would eventually allow me to purchase some of their technology."

> 				log "Factions" "Successors" "The Successors are governed by a group of six High Houses who operate mostly independently from each other. The Houses are organized into two large alliances called the Old and New Houses, which have historically been in tension with each other. The Houses cooperate in council with their allies and the other Houses in general."

Reviewer 6: I don't think this is explained in the mission?

			`	Riijaaora gestures to a small wooden pin affixed to the outer layer of their clothing. "We keep a number of these devices, full of sound and language, but very few for thine own tongue. This one... 'tis an art piece, issued many years past for a performance of thine own Shakespeare's Tempest so that our noble Predecessors could experience the work in its truest form. It is now thy promised gift."`
			`	Riijaaora plucks the device from their outfit and offers it to you.`
			choice
				`	(Take it.)`
			`	The device is made of carved wood, or at least something that appears like it, depicting in well-varnished grain a sailing ship washed against rocks. As you pin it on your jacket, the ship begins to gently rock in place. Riijaaora turns away from you to speak to Sieasej. "And think not, fair citizen, that thine own contributions shall go without recognition. Thou shalt be rewarded, too, for thy part in this." Riijaaora gestures for another officer to escort them away, though not before Sieasej offers you a short splay of the arms in goodbye.`

> 			`	An officer approaches you to offer you a data-sphere. "Within this," Riijaaora says, "lies knowledge of the history of most worlds within our space. Combined with thy newfound skill of tongues, thou should'st find our worlds and cities more revealing to thy gaze and more fruitful for thine efforts. Thou shall'st be permitted to looks upon our creations and our vessels. Should'st the prospect interest thee, we're sure that the other Houses in their council would agree to extend thee purchase rights, provided thou displays to them thy beneficence and utility. There're many minor tasks which need completion on our worlds; no doubt thy prompt assistance would help in such ingratiation to the others." Riijaaora wiggles. "'Tis all, unless thou hast more questions thou needeth us t'resolve?"`

Reviewer 6: Does Riijaaora have a second device to speak with?

Author: No. The intention for this section is that once First Last takes the device, it now functions to translate all the native Successor language they hear (which Riijaaora is speaking) to FL's own language. I could make this clearer with a sensory description of surrounding sounds feeling "different" once FL takes it, e.g. `...rock in place, and you feel a gentle tingling in your ear canals.` or something like that.

Reviewer 6: Ah, that makes sense.

					goto end
				`	"Will being your ally bring me into conflict with the other Houses?"`
					to display
						not "Successors: others"
					goto others
				`	"What are you going to do to the animal?"`
					to display
						not "Successors: animal"
					goto animal
			label others
			action
				set "Successors: others"
			`	Riijaaora's skin blotches with colors. "Surely not, we think. For none would make so bold a slight as t'endanger the vassal of another. Thou may'st travel 'tween our stars 'thout fear."`
				goto questions
			label animal
			action
				set "Successors: animal"
			`	"'Tis none of thy concern, fair captain. Though, if thou fear'st cruelty on our part, know well thy gift shall be by us well-kept and well-preserved. Fear not for it, and quell thy doubtful heart."`
				goto questions
			label end
			action
				clear "Successors: animal"
				clear "Successors: others"
			`	"Good. We bid thee good farewells, then, and good tidings in thy travels." Riijaaora and the other scions turn around to face their backs towards you and the officers who escorted you in escort you back out to your ship through the various sculptural passages of Sioeora's complex.`
			label exit

> substitutions

Reviewer 6: This should also be moved into the mission.

			`	"Captain <first> <last>," they say. Their voice sounds strange, like their words are being carried from a great distance on the wind; the sounds ripple and bend as they speak them.`
			choice
				`	"That's right."`
				`	"Who are you?"`
			`	"Hmm. Good. I'm an envoy of the noble scions of House Kaatrij. I attest this myself as Saajret, of Somber Sejra." Saajret's skin erups with green splashes as they say this. "Thy lib'ral patience is well-appreciated." They splay their arms in the same manner as when you were first greeted by the Successors, and the two others behind them do so as well. "Ah," they continue, "but thou surely must be curious as to why we've called thee here. Thine actions, knowing or not, have benefitted this noble House, and 'twould be craven fools to fail to recognize that fact. Prithee understand thee now that Raaqa-Uur-Kaav was our world and not Sioeora's. So too, the Kijran, ours, were never theirs to hold or grasp for."`
			choice
				`	"So the Kijran belong to you?"`
					goto kijran

> 				`	"How is it that you can communicate with me?"`

Reviewer 6: ```suggestion
				`	"How is it that you can communicate with me?"`
					to display
						not "language: Successor"
```

					goto communicate
			label questions
			choice
				`	"How is it that you can communicate with me?`
					to display
						not "language: Successor"
					goto communicate
				`	"What is next?"`
					to display
						has "language: Successor"
					goto end
				`	"Will your supporting me cause conflict with House Sioeora?"`
					to display
						has "language: Successor"
						not "Successors: sioeora"
					goto sioeora
				`	"Can you tell me more about yourself?"`
					to display
						has "language: Successor"
						not "Successors: saajret"
					goto saajret

> 				`	"You mentioned earlier that the Kijran 'belong' to you?"`

Reviewer 6: ```suggestion
				`	"You mentioned earlier that the Kijran 'belong' to you?"`
					to display
						not "Successors: kijran"
```

				"reputation: House Chydiyi" += 5
				"reputation: House Myurej" += 5
				"reputation: House Aqrabe" += 10
				"reputation: House Kaatrij" += 10
				"reputation: House Seineq" += 10
				"reputation: People's Houses" += 10

mission "Successors: Kaatrij Monoglot Prompt"
	landing
	invisible
	to offer
		has "event: successors: released kijra-pet wait"
		not "failed: Successors: First Contact 3"
		and
			not "language: Hai"
			not "language: Coalition"
		random < 30
	on offer
		conversation
			`Shortly after you set the <ship> down on <origin>, you notice a faint trilling coming from your cabin on board. After rummaging through some of your belongings, you discover that it is coming from a Successor data-sphere, which is vibrating intensely. You pick it up, and it projects a message in a strange-looking alien script that you unfortunately do not recognize. If you ever acquired some alien translation equipment, you might be able to decode it.`
			decline

substitutions
	"<successor patron>" "Sioeora"
	"<successor patron>" "Kaatrij"
		has "completed: Successors: Kaatrij First Contact 1"
	"<successor opponent>" "Kaatrij"
	"<successor opponent>" "Sioeora"

> 		has "completed: Successors: Kaatrij First Contact 1"

Reviewer 6: This should be within the mission as well.

			`	Sieasej brings you to a concierge of sorts, who eventually informs you that you should make your way to one of the smaller rooms branching off from the lobby. The doors open to reveal a garden-like area, filled with large long-leafed plants surrounding a centrally-located depression half-filled with water. Several Successors, many well-dressed like before, are gathered in the pool; you can't help but wince a little as the frigid water floods your boots.`
			branch kaatrij
				has "completed: Successors: Kaatrij First Contact 1"
			`	After a few minutes and other Successors have trickled in, the meeting begins in earnest. One of the Successors moves to the center of the pool and addresses you directly.`
			`	"Captain. House Sioeora's supported thee with the gift of our device of tongues, and we continue t'recognize thy potential as an ally. We wish to further 'ppreciate the good that thou hast done upon our worlds, and we propose thou be awarded purchase rights for our petty technologies."`
			`	That representative then returns to the outer rim of the circle and another takes the center, this time facing the representative who just spoke.`

> 			`	"House Kaatrij. House Chydiyi well sees thy perspicacity and, too, the value demonstrated by the captain, and so we support thy just proposal. The captain's proven themself t'be a benevolent traveler in our space, for whom we've every desire to make an ever more positive and useful element."`

Reviewer 6: ```suggestion
			`	"House Sioeora. House Chydiyi well sees thy perspicacity and, too, the value demonstrated by the captain, and so we support thy just proposal. The captain's proven themself t'be a benevolent traveler in our space, for whom we've every desire to make an ever more positive and useful element."`
```

	on offer
		conversation
			`Compared to most of the spaceports you have been to in Successor space, this one is dingy, crowded, and uncomfortable. Most of the Successors you see look equally dirty and unhappy, and the majority of the few ships you've noticed landing are painted in a different, darker shade of red.`
			`	Would you like to ask someone here about this place?`
			choice
				`	(Yes.)`
				`	(Not yet.)`
					defer
			`	You approach one of the crew members disembarking from one of the red-painted ships, which, as you draw closer, you see bears several large burns and scars.`
			`	"Excuse me, I was wondering if you could tell me about this place. It seems different to the rest of Successor space I've been through."`
			`	The Successor looks over at the spaceport for a moment before grunting quietly and deciding to reply. Their voice is lower and darker than that of most Successors you have heard, and their skin changes color and texture rapidly, almost with every word.`
			`	"Thou art the alien vis'tor brought by the Old Houses, art thou not? Their 'noble guest' whom they parade around with such delight?"`
			choice
				`	"Yes."`
				`	"You could say so."`

> 			`	Despite the accusation of being "paraded around," the Successor grunts again at your response; with approval or disgust you cannot tell. They lean against one of the cargo containers on the spaceport floor and continue,`

Reviewer 6: ```suggestion
			`	Despite the accusation of being "paraded around," the Successor grunts again at your response; with approval or disgust you cannot tell. They lean against one of the cargo containers on the spaceport floor and continue.`
```
